crates/epaint/src/image.rs

```diff
@@ -101,6 +101,21 @@ impl ColorImage {
         Self { size, pixels }
     }
 
+    /// Create a [`ColorImage`] from flat un-multiplied RGB data.
+    ///
+    /// This is what you want to use after having loaded an image file (and if
+    /// you are ignoring the alpha channel - considering it to always be 0xff)
+    ///
+    /// Panics if `size[0] * size[1] * 3 != rgba.len()`.
+    pub fn from_rgb_unmultiplied(size: [usize; 2], rgb: &[u8]) -> Self {
```

Comment:
```suggestion
    pub fn from_rgb(size: [usize; 2], rgb: &[u8]) -> Self {
```

if you only have rgb, there is no alpha for it to have been multiplied with